pkg/controller/cronjob/utils.go

```diff
@@ -86,19 +86,18 @@ func groupJobsByParent(js []batchv1.Job) map[types.UID][]batchv1.Job {
 	return jobsBySj
 }
 
-// getRecentUnmetScheduleTimes gets a slice of times (from oldest to latest) that have passed when a Job should have started but did not.
-//
-// If there are too many (>100) unstarted times, just give up and return an empty slice.
-// If there were missed times prior to the last known start time, then those are not returned.
-func getRecentUnmetScheduleTimes(sj batchv1beta1.CronJob, now time.Time) ([]time.Time, error) {
-	starts := []time.Time{}
+// getRecentUnmetScheduleTime returns some time that has passed when a Job should have started but did not.
```

Comment:
I'd prefer something more concrete in the docs than just `some time has passed`.